frontend/src/router/components/protected-route.tsx

```diff
@@ -0,0 +1,16 @@
+import { RouterOutlet } from '~/bundles/common/components/components.js';
+import { AppRoute } from '~/bundles/common/enums/app-route.enum.js';
```

Comment:
It would be best if you import AppRoute from enums/enums also here

Reply:
import from root file